DocumentDataAPI/DocumentDataAPI/Controllers/DataSourceController.cs

```diff
@@ -6,14 +6,14 @@
 namespace DocumentDataAPI.Controllers;
 
 [ApiController]
-[Route(RoutePrefixHelper.Prefix + "/sources")]
+[Route(RoutePrefixHelper.Prefix + "/data-sources")]
 [Produces(MediaTypeNames.Application.Json)]
-public class SourceController : ControllerBase
+public class DataSourceController : ControllerBase
```

Comment:
Jeg er sgu ikke helt sikker på, at jeg er solgt på navnet DataSource.

Reply:
Jeg er totalt enig, men det er de ændringer vi blev enige om at tilføje fra gruppe A

Reply:
Spørgsmålet vi bør stille os selv, er så om vi bestemmer navnene og overrider deres request, da vi laver databasen, eller om vi skal tilfredsstille dem? Det er no hard feelings til dem, men jeg synes bare ikke til tilføjer andet end en smule ekstra 
![Confucius](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fi2.wp.com%2Fgosouth.co.za%2Fwp-content%2Fuploads%2F2016%2F12%2Fconfucius-e1483374884833.jpg%3Ffit%3D1200%252C978%26ssl%3D1&f=1&nofb=1&ipt=75ad13aa68e92c9b7f6fae1bcf93f390a841854966153c92f1b141727d65f6de&ipo=images).

Reply:
Vi kan godt gå tilbage til source for min skyld. Grunden til at de ikke kunne lide det var at det kunne forstås som kildehenvisninger, så data source specificerer hvad det handler om